Where_Is_My_Wife/Assets/_Main/Scripts/Animation/AnimationController.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using UniRx;
+using UnityEngine;
+using WhereIsMyWife.Managers;
+using Zenject;
+
+namespace WhereIsMyWife.Controllers
+{
+    public class AnimationController : MonoBehaviour
```

Comment:
Change name to `PlayerAnimationController`

Reply:
389b7d6bb19af30a9f86222a4b881b7dff04abef